util/unifyfs/src/unifyfs.c

```diff
@@ -92,6 +96,9 @@ static char* usage_str =
     "Common options:\n"
     "  -d, --debug               enable debug output\n"
     "  -h, --help                print usage\n"
+    "  -t, --timeout=<time>      [OPTIONAL] timeout value\n"
```

Comment:
This option is already on line 108 as an option for the start command. We need it twice?